master/usage/configuration/node.md

```diff
@@ -0,0 +1,95 @@
+---
+title: Configuring a node
+---
+
+Each Calico node has key routing configuration that may be specified in a number of
+different ways.  In particular, this includes the IPv4 and IPv6 addresses and 
+subnets used for routing, and the BGP AS Number (which generally only needs to 
+be changed if you are explicitly configuring BGP peerings with your Calico nodes).
+
+To configure the routing parameters on your node, you have the options of:
+-  Set appropriate flags on the `calicoctl node run` command
+-  Set appropriate environments for the `calico/node` container
+-  Configure the node resource using `calicoctl`
+
+> Note that for nodes that were configured prior to v2.1, the subnet information 
+> configured on those nodes will be set to /32 for IPv4 or /128 for IPv6.  If you are
+> planning on using the `cross-subnet IP-in-IP` feature then you must update your node
+> configuration to have the correct subnet information for each node.
```

Comment:
The way it reads to me right now is that this information must be manually set.  If there are autodetection modes that will set it or not set it I think that should be outlined.  @robbrockbank  From what you said, it sounds like autodetection only fixes the subnet if IP=autodetect is explicitly set, I think that is good info to include.  If that is not the case and any autodetection will update the configured subnet then maybe this note should just comment on if the previous config explicitly set the IP then it must be updated.